module1/homework/grayscaleImages/test.cpp

```diff
@@ -3,7 +3,7 @@
 #include <utility>  // for std::pair<>
 #include <vector>
 
-// TODO: include
+#include "compression.hpp"// TODO: include
```

Comment:
You can remove vector include here since it's in the header.

Reply:
I will leave it, since the code was not mine :)

Reply:
Never mind! I wasn't aware of "Include-what-you-use" rule,which tells that you include everything you are using even if it would be included by another file. Sorry!